The flag added here avoids a warning on some version of AM.
Not much suitable for community commit because it is AM-version dependent.

Taken from solaris-userland.
diff --git a/Makefile.am b/Makefile.am
index b3a8739..c12d8f0 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
+AUTOMAKE_OPTIONS = subdir-objects
 
 pkgconfdir = $(libdir)/pkgconfig
 pkgconf_DATA = \
